index.html

```diff
@@ -25,23 +25,32 @@
   <body>
     <main>
       <header>
-        <nav>
+        <nav id="navId">
           <div class="logo">
-            <h1>My Logo</h1>
+            <h1 class="logo-text">My Logo</h1>
           </div>
-          <div class="button">
-            <i class="fa fa-bars fa-2x"></i>
+          <div
+            class="button"
+            id="mobile-menu"
+            class="mobile-menu-class"
+            onClick="toggleMenu();"
+          >
+            <i id="hm-menuIcon" class="fa fa-bars fa-2x"></i>
```

Comment:
- [ ] Update the weight/thickness of the X, in order to match the design
Yours: 
![image](https://user-images.githubusercontent.com/62705947/194943416-f05455b8-3696-4167-806d-df006489015a.png)

Design: 
![image](https://user-images.githubusercontent.com/62705947/194943477-ad4454ff-1115-435c-90c8-103149dbc7e6.png)